scripts/nlp_api_master.py

```diff
@@ -996,6 +1223,7 @@
     "title": "KerasNLP",
     "toc": True,
     "children": [
+        BASE_CLASSES,
```

Comment:
I'd recommend having the base classes towards the end, since they're less likely to be the first thing people want to learn about (that would be the models)

Reply:
Maybe second? I think more people would care about these than the built-in layers.